invenio_app_ils/records/jsonresolver/item.py

```diff
@@ -16,4 +16,7 @@
 )
 def loan_for_item_resolver(pid_value):
     """Return the circulation status for the given item."""
-    return get_loan_for_item(pid_value) or {}
+    loan = get_loan_for_item(pid_value) or {}
```

Comment:
We have to chnage this `jsonresolver` to the other implementation, to avoid to hardcode `localhost`.
Ticket?

Reply:
Ticket https://github.com/inveniosoftware/invenio-app-ils/issues/134 :)